data/registry/instrumentation-php-psr3.yml

```
@@ -0,0 +1,15 @@
+title: OpenTelemetry PSR-3 auto-instrumentation
+registryType: instrumentation
+isThirdParty: false
+language: php
+tags:
+  - psr3
+  - instrumentation
+  - php
+repo: https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/src/Instrumentation/Psr3
+license: Apache 2.0
+description:
+  Auto-instrumentation hooks are registered via composer, and automatically
+  inject trace id and span id into log message context of any psr3 logger.
```

Should there be a textlint entry for "PSR-3"?

```suggestion
  Auto-instrumentation hooks are registered via composer, and automatically
  inject trace ID and span ID into log message context of any PSR-3 logger.
```

---

good call out!

done & rule added

---

🙌🏻